src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -193,7 +187,6 @@ public override string ReadLine()
         /// <exception cref="PipelineStoppedException">
         /// If Ctrl-C is entered by user
         /// </exception>
-
```

Comment:
This type of change is not a code redundancy change.  Please don't combine.  The PR is too large to review already.

Reply:
It is a redundant empty line. I can split into seperate PR.